### PR TITLE
implemented whats new dialog

### LINKED ISSUE #2247: What's new dialog upon 1st launch after app is update
https://github.com/AlphaWallet/alpha-wallet-ios/issues/3462

Zeplin:
https://zpl.io/VQ3ZgDx

Wallet-News

### PR DESCRIPTION
**Pull request by mpaschenko**
Closes #2247

### CODE REVIEW

**Comment by JamesSmartCell** [comment]
@mpaschenko looks great, nice work.

File: app/src/main/res/values-zh/strings.xml [hunk 1]
```diff
@@ -826,4 +826,5 @@
     <string name="summary_change_24h">%1$s (%2$s) 今天</string>
     <string name="performance">Performance</string>
     <string name="token_name_hint">Token Name</string>
+    <string name="whats_new">What\'s new?</string>
```
Comment (seabornlee) [inline app/src/main/res/values-zh/strings.xml:829]: The Chinese translation: 新功能

Comment (JamesSmartCell) [reply app/src/main/res/values-zh/strings.xml:829]: @seabornlee can you update the PR with this?

Comment (seabornlee) [reply app/src/main/res/values-zh/strings.xml:829]: > @seabornlee can you update the PR with this?
Sure.

Comment (seabornlee) [reply app/src/main/res/values-zh/strings.xml:829]: Sure.

**Review by JamesSmartCell — APPROVED** [review]

**Pull request merged**